ArkKit/ark-ui-kit-renderer/view-components/input/UIKitButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UIKitButton.swift
+//  ArkKit
+//
+//  Created by En Rong on 16/3/24.
+//
+
+import UIKit
+
+class UIKitButton: UIView, UIKitRenderable, TapRenderable {
+    var onTapDelegate: TapDelegate?
+
+    init() {
+        let frame = CGRect(x: 0, y: 0, width: 100, height: 50)
+        super.init(frame: frame)
+
+        // TODO: Update UIButton styling
+        let button = UIButton()
+        button.addTarget(self, action: #selector(handleTap), for: .touchUpInside)
```

Comment:
Think there has to be an additional `self.addSubview(button)` for the `UIButton` component to render into the `UIView`. I'd do this separately to complete the `UIKit` implementations together with managing the max pan-ability of the `UIKitJoystick`.